internal/testplanet/planet.go

```diff
@@ -137,11 +138,17 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 			}))
 
 		go func(n *Node) {
+			if err := n.Discovery.Bootstrap(ctx); err != nil {
```

Comment:
What's the difference between Bootstrap and Refresh? Why does it keep calling Bootstrap.

Reply:
This was due to a bug that I fixed. I had it pushed up like this to demonstrate the bug.